server/service/src/sync/translations/special/oms_object_fields.rs

```diff
@@ -0,0 +1,90 @@
+use serde::{Deserialize, Serialize};
+
+use crate::sync::sync_serde::object_fields_as_option;
+
```

Comment:
This file is generic handling of all possible null cases including empty string by the new deserialiser